src/components/visualizations/chart/highChartsCreators.ts

```diff
@@ -48,7 +48,7 @@ export function getHighchartsOptions(chartOptions: IChartOptions, drillConfig: a
     return merge(
         {},
         getCommonConfiguration(chartOptions, drillConfig),
-        getConfigurationByType(),
+        getConfigurationByType.call(null, config),
```

Comment:
If changing to `getConfigurationByType(config)` it will cause typescript error since `getConfigurationByType` is a mapping function and can receive no parameter